remoting/WFLY-19992-Enable-configuration-of-selected-XNIO-Remoting-parameters-for-remoting-objects-using-in-both-inbound-and-outbound-communication.adoc

```diff
@@ -0,0 +1,128 @@
+---
+categories:
+- core
+- remoting
+stability-level:
+default
+issue:
+https://github.com/wildfly/wildfly-proposals/issues/667
+feature-team:
+ developer:
+ tadamski
+ sme:
+  -
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= <INSERT TITLE HERE>
+:author:            Tomasz Adamski
+:email:             tadamski@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+During analysis of various support cases we found out that there is no possibility to configure XNIO/Remoting options for connections that use remoting protocol (f.e. EJB). It it worth emphasizing that this issue doesn't deal merely with the lack of options parsing that has to be implemented. There are many objects that are created by remoting f.e. Endpoints or Channels, and, what is important in the context of this issue, they can be used in both inbound and outbound scenario. So the goal of this issue is to design and document the elegant way of configuring remoting options.
```

Comment:
We can configure certain XNIO/Remoting options, but we cannot generally configure all XNIO/Remoting options.